rest/doc_api.go

```diff
@@ -167,6 +167,7 @@ func (h *handler) handleGetDocReplicator2(docid, revid string) error {
 			return err
 		}
 	}
+	h.db.DbStats.StatsDatabase().Add(base.StatKeyNumDocReadsRest, 1)
```

Comment:
Incrementing the stat after response.Write for consistency with handleGetDoc would be preferable, I think.